packages/desktop-client/src/components/table.tsx

```diff
@@ -796,15 +797,33 @@ export function TableHeader({
   );
 }
 
-export function SelectedItemsButton({ name, items, onSelect }) {
-  const selectedItems = useSelectedItems();
+type SelectedItemsButtonNames = 'transactions' | 'schedules';
+
+type SelectedItemsButtonProps<T extends MenuItem = MenuItem> = {
+  name: SelectedItemsButtonNames;
+  items: Array<T | typeof Menu.line>;
+  onSelect: (name: string, items: Array<T | typeof Menu.line>) => void;
+};
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏nice